content/en/docs/concepts/architecture/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,79 @@
+---
+reviewers:
+- jpbetz
+title: Unknown Version Interoperability Proxy
+content_type: concept
+weight: 220
+---
+
+<!-- overview -->
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that lets one
+{{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}
+proxy a resource request to a different _peer_ API server. This is useful when there are multiple
+API servers running different versions of Kubernetes in one cluster (for example, during a long-lived
+rollout to a new release of Kubernetes).
+
+This mechanism is called the _Unknown Version Interoperability Proxy_.
+## Enabling the Unknown Version Interoperability Proxy
+The Unknown Version Interoperability Proxy feature is controlled by a
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+and is not enabled by default. You can activate Unknown Version Interoperability Proxy by enabling
+the feature gate named `UnknownVersionInteroperabilityProxy` when you start the
+{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}:
+
+```shell
+kube-apiserver \
+--feature-gates=UnknownVersionInteroperabilityProxy=true \
+# required flag for this feature
+--peer-ca-file=`ca bundle to verify serving certs of kube-apiserver`
+# optional flags for this feature
+--peer-advertise-ip=`ip of this kube-apiserver that should be used by peers to proxy requests`
+--peer-advertise-port=`port of this kube-apiserver that should be used by peers to proxy requests`
```

Comment:
Would it be more helpful to readers to show the flags with valid inputs?

Should we be linking to/from:

- https://kubernetes.io/docs/tasks/administer-cluster/certificates/
- https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/
- https://kubernetes.io/docs/tasks/administer-cluster/cluster-upgrade/ 

or similar?

Reply:
Did you mean specifying an example path for the cert files as inputs?

I added a link to [proxy cert setup](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-aggregation-layer/#kubernetes-apiserver-client-authentication) for users to refer to better understand the --proxy-* and the --requestheader-client-ca flags

Reply:
Linking *from* https://kubernetes.io/docs/tasks/administer-cluster/cluster-upgrade/ makes sense to me. Not sure about the other two..